v2/cli/init_command.go

```diff
@@ -0,0 +1,256 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/brigadecore/brigade/sdk/v2/core"
+	"github.com/brigadecore/brigade/v2/internal/file"
+	"github.com/urfave/cli/v2"
+)
+
+var initCommand = &cli.Command{
+	Name:  "init",
+	Usage: "Initialize a new Brigade project",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:    flagID,
+			Aliases: []string{"i"},
+			Usage: "Name of the Brigade project to initialize " +
+				"(required)",
+			Required: true,
+		},
+		&cli.StringFlag{
+			Name:    flagLanguage,
+			Aliases: []string{"l"},
+			Usage: "Specify the scripting language to use for handling events" +
+				" -- JavaScript (js) or TypeScript (ts)",
+			Value: "ts",
+		},
+		&cli.StringFlag{
+			Name:    flagGit,
+			Aliases: []string{"g"},
+			Usage: "Optionally specify a git repository where the" +
+				" event handling script can be found",
+		},
+	},
+	Action: initialize,
+}
+
+// nolint: gocyclo
+func initialize(c *cli.Context) error {
+
+	// Initialize fields to fill in the template yaml files
+	fields := struct {
+		ProjectID   string
+		Language    string
+		GitCloneURL string
+		Script      string
+	}{
+		c.String(flagID),
+		strings.ToLower(c.String(flagLanguage)),
+		c.String(flagGit),
+		"",
```

Comment:
You should always include keys when initializing fields in a struct type. All it would take is for the order of some fields in lines 49 - 52 to be changed and we'd be in deep trouble. If we're in the habit of always using keys, that same change would have no impact.